### PR TITLE
HERON Using SyntheticHistory/TSA ROMs

### LINKED ISSUE #323: [TASK] HERON with New SyntheticHistory/TSA ROMs
--------
Issue Description
--------

**Is your feature request related to a problem? Please describe.**
HERON should switch to using Synthetic History ROMs from RAVEN to replace the old ARMA ROMs. 

**Describe the solution you'd like**
We should add new versions of the same Sine and NYISO but trained using the Synthetic History and TSA modules in RAVEN. Existing regression tests should use these new ROMs to check that they are compatible. 

**Describe alternatives you've considered**

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [ ] 1. Is it tagged with a type: defect or task?
- [ ] 2. Is it tagged with a priority: critical, normal or minor?
- [ ] 3. If it will impact requirements or requirements tests, is it tagged with requirements?
- [ ] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [ ] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [x] 1. If the issue is a defect, is the defect fixed?
- [x] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [x] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [x] 4. If the issue is a defect, does it impact the latest release branch? If yes, is there any issue tagged with release (create if needed)?
- [x] 5. If the issue is being closed without a pull request, has an explanation of why it is being closed been provided?

### PR DESCRIPTION
**Pull request by GabrielSoto-INL**
--------
Pull Request Description
--------
##### What issue does this change request address?
#323 

##### What are the significant changes in functionality due to this change request?
- New versions of the Sine, SineHour, Sine30Yr, and NYISO 2yr ROMs have been added that are trained using the `SyntheticHistory` module in RAVEN. 

- XML files for each are also added in a new `TSA` subdirectory in the `tests` folder. An additional XML for the NYISO 20yr is also added but the new pickle file is omitted due to size constraints (~120MB). 

- Regolding some regression tests due to small differences in new TSA algorithms within `SyntheticHistory` compared with the old `SupervisedLearning.ARMA` module

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/HERON/wiki/Code-Standards) for details.
- [x] 4. Automated Tests should pass.
- [x] 5. If significant functionality is added, there must be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large tes.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added, the the analytic documentation must be updated/added.
- [x] 9. If any test used as a basis for documentation examples have been changed, the associated documentation must be reviewed and assured the text matches the example.

### CODE REVIEW

File: tests/workshop/wind/gold/OptimizationSolutions/NPP_Wind/opt_heron_input.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <Case name="wind_coarse_opt">
     <mode>opt</mode>
     <!-- <mode>sweep</mode> -->
-    <debug/>
+    <!-- <debug/> -->
```
Comment (PaulTalbot-INL) [inline tests/workshop/wind/gold/OptimizationSolutions/NPP_Wind/opt_heron_input.xml:16]: does this still run the right tests when it's swapped off debug mode?

**Review by PaulTalbot-INL — COMMENTED** [review]
Changes look okay to me, just one comment to consider

**Comment by PaulTalbot-INL** [comment]
Checklist passes, waiting on one comment clarification before merging.

File: tests/workshop/wind/gold/OptimizationSolutions/NPP_Wind/opt_heron_input.xml (hunk 1 above)
Comment (GabrielSoto-INL) [reply tests/workshop/wind/gold/OptimizationSolutions/NPP_Wind/opt_heron_input.xml:16]: yes, the debug mode line should have been commented out in the first place (artifact from copying over/live demo). 

also some clarification: these aren't being tested with rook. they _could_ be tested, but the tests would take a very long time. I had meant for this to be resource for people who wanted to run the workshop case in opt mode for reference/comparison.

Comment (PaulTalbot-INL) [reply tests/workshop/wind/gold/OptimizationSolutions/NPP_Wind/opt_heron_input.xml:16]: Okay, good to know. Would they be covered by heavy tests running, or is there no rook hook for them?

**Review by PaulTalbot-INL — APPROVED** [review]

**Pull request merged**